Resources/Prototypes/Objectives/traitor.yml

```diff
@@ -111,6 +111,20 @@
     # if ce flies a shittle to centcom you better find a way onto it
     requireDead: true
 
+- type: entity
+  parent: [BaseTraitorObjective, BaseKillObjective]
+  id: KillRandomTraitorObjective
+  description: We have reason to beleve that they have become unloyal to the syndicate and need them elimninated.
```

Comment:
believe, eliminated

Reply:
Yeah I'm bad at spelling thanks for the catch!